.github/workflows/docker-image.yml

```
@@ -9,19 +9,37 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
+
+    - uses: hadolint/hadolint-action@v3.1.0
+      name: Lint Dockerfile
+      with:
+        dockerfile: Dockerfile
```

we could all the ignore here
```suggestion
        dockerfile: Dockerfile
        ignore: DL3007,DL3018
```

---

@Azanul Add these ignore to all 3 hadolint